crates/typst/src/layout/repeat.rs

```diff
@@ -42,7 +42,7 @@ pub struct RepeatElem {
 
     /// Whether to increase the gap between instances to completely fill the
     /// available space.
-    #[default(true)]
+    #[default(true)] // Maybe disable it for `gap to work as expected?`
```

Comment:
Not sure how this counts as a doc change (I'd rather have this in a separate PR). You could open (or link to) an issue demonstrating how the current behavior is broken (if so).

Reply:
See the OP:

> I created a PR with all the polished and fixed stuff (plus nitpicks and **questions/proposals**)

Are you sure PR is OK instead of an issue? I wanted to have a (small) discussion about this, and also I didn't yet think of a code change (changing `true` to `false` might not be enough), so for a PR I need to come up with a proposed change.

Reply:
Open an issue, please.